src/app/vault/vault.component.ts

```diff
@@ -609,4 +619,8 @@ export class VaultComponent implements OnInit, OnDestroy {
         this.addOrganizationId = null;
         this.addCollectionIds = null;
     }
+
+    private selectedCipher() {
```

Comment:
Is there an easiest way to access the currently selected cipher?

Reply:
Seems ok, but this can return null, which you don't account for when calling this function

Reply:
Changed this.